packages/apps/tasks/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "tasks",
+  "version": "0.1.0",
+  "description": "tasks - a DXOS application",
+  "private": true,
+  "scripts": {
+    "build": "tsc --noEmit && vite build",
+    "deploy": "dx app publish",
+    "preview": "vite preview",
+    "serve": "vite",
+    "storybook": "start-storybook -p 9009 --no-open",
+    "prebuild": "dxtype src/proto/schema.proto src/proto/gen/schema.ts"
+  },
+  "dependencies": {
+    "@dxos/client": "workspace:*",
+    "@dxos/config": "workspace:*",
+    "@dxos/vault": "workspace:*",
+    "@dxos/react-client": "workspace:*",
+    "@dxos/react-shell": "workspace:*",
+    "@dxos/aurora": "workspace:*",
+    "@dxos/echo-schema": "workspace:*",
```

Comment:
did I do the workspace thing right?

Reply:
yeah, pnpm replaces those on publish https://pnpm.io/workspaces#publishing-workspace-packages